### ISSUE TITLE
Checkbox without label

### ISSUE THREAD

**Issue by mbrpf** [markocupic/contao-altcha-antispam#16]
The label of the checkbox of the altcha has a diffferent for-value than the checkbox id.
Testing a form with WAVE accessibility test tool brought this up as an error (missing form label)
